check_warning_limit.py

```diff
@@ -0,0 +1,63 @@
+from Print import print_warning,German_pring_warning
+
+def warning_function(value,min,max,function_name):
+    warning_value = Warning_value_finder(min, max)
+    feedback = warning_check_function(value,min,warning_value[0],warning_value[1],max)
+    if(feedback == True):
+        print_warning(function_name)
```

Comment:
function_name value is in English for both English and German language, whereas it should have been German string for German language.